### PR TITLE
Submission auth

### PR DESCRIPTION
**Pull request by keegansmith21**
I forgot what i did but this works now

### CODE REVIEW

File: database/submit_to_database.py [hunk 1]
```diff
@@ -570,95 +679,35 @@ def flux_cal_and_submit(time_detection, time_obs, metadata, bestprof_data,
...
-            pulsar = args.pulsar
-        dm, period = get_pulsar_dm_p(pulsar)
-        period_uncer = float(period) / 10.**10 #assuming it's miniscule
-        if args.stop and args.start:
-            time_detection = args.stop - args.start
-        else:
-            logger.error("Please use --start and --stop for ascii files. Exiting")
-            sys.exit(1)
-        bestprof_data = [obsid, pulsar, dm, period, period_uncer, args.start,
-                         time_detection, profile, num_bins]
```
Comment (NickSwainston) [inline database/submit_to_database.py:597]: You've removed the ascii option here and looks like you haven't added an option to process them at line 707

File: database/submit_to_database.py [hunk 2]
```diff
@@ -508,12 +633,7 @@ def flux_cal_and_submit(time_detection, time_obs, metadata, bestprof_data,
...
-    dspsrargs.add_argument('-f','--fits_files',type=str,help='The fits files location to be used in any Detection Calculation processing. Recommended to end in *.fits and surrounded by quotation marks.', default = "/group/mwavcs/vcs/${obsid}/fits/*fits")
-    dspsrargs.add_argument('--u_archive', action='store_true',help='Used to create an archive file using DSPSR.')
-    dspsrargs.add_argument('--u_single_pulse_series', action='store_true',help='Used to create a single pulse archive using DSPSR.')
-    dspsrargs.add_argument('--u_ppps', action='store_true', help="Used to create a Presto Prepfold PostScript file using PRESTO")
-    dspsrargs.add_argument('--u_ippd', action='store_true', help="Used to create an Intergrated Pulse Profile (sometimes called a pulse profile) using DSPSR")
-    dspsrargs.add_argument('--u_waterfall', action='store_true', help="Used to create a waterfall plot of pulse phase vs frequency using DSPSR.")
+    dspsrargs.add_argument('-f','--fits_files',type=str,help='The fits files location to be used in any Detection Calculation processing. Recommended to end in *.fits and surrounded by quotation marks.', default = "/group/mwaops/vcs/${obsid}/fits/*fits")
```
Comment (NickSwainston) [inline database/submit_to_database.py:636]: I don't think you need these either now that you've removed the dspsr options

File: database/submit_to_database.py (hunk 1 above)
Comment (keegansmith21) [reply database/submit_to_database.py:597]: Fixed this but please double check my code

File: database/submit_to_database.py [hunk 3]
```diff
@@ -702,9 +694,20 @@ def flux_cal_and_submit(time_obs, metadata, bestprof_data,
     else:
         obstype = 2
 
-    if args.bestprof or args.ascii:
+    if args.bestprof:
         #Does the flux calculation and submits the results to the MWA pulsar database
         bestprof_data = prof_utils.get_from_bestprof(args.bestprof)
+    elif args.ascii:
+        if not args.stop or not args.start:
+            logger.error("Please suppl both start and stop times of the detection for ascii files")
```
Comment (NickSwainston) [inline database/submit_to_database.py:702]: supply

Comment (keegansmith21) [reply database/submit_to_database.py:702]: >:(

**Pull request merged**